scripts/common/install-latest-librdkafka.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#### Install latest librdkafka, right from the GIT repo.
+
+### Define the version of kafka we are pinning to.
+kafka_version=1.5.0
+
+
+### Install librdkafka
+## Install dependencies
+apk add --no-cache libressl-dev || exit 2
```

Comment:
Aren't all of these dependencies already included in the [hooks/build](https://github.com/better/dockerimages/blob/master/hooks/build) dependencies?

If so, we don't need to reinstall them here!

Reply:
I don't assume that they are. Since rerunning the install is a nop, these are really just comments saying "ensure that the stuff I need is installed".